lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3173,11 +3247,11 @@ class ConvertTorchToTosa : public ConvertTorchToTosaBase<ConvertTorchToTosa> {
                                            tosa::AvgPool2dOp);
 #undef INSERT_ADAPTIVE_POOLING_ATEMOP_PATTERN
 
-target.addIllegalOp<AtenMaxPool2dOp>();
-patterns.add<ConvertAtenMaxPool2dOp>(typeConverter, context);
+    target.addIllegalOp<AtenMaxPool2dOp>();
```

Comment:
nit: why the indent?

Reply:
Yeah that's weird. Looks like a preexisting condition that's not covered by my insurance here though. I'm surprised the clang-format didn't fix it, or maybe it's what broke it.

Reply:
Don't worry about it. Yeah, fine to leave it.